x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/create_connector_flyout/header.tsx

```diff
@@ -63,23 +61,14 @@ const FlyoutHeaderComponent: React.FC<Props> = ({
                     </h3>
                   </EuiTitle>
                 </EuiFlexItem>
-                {actionTypeName
-                  ? isBeta && (
-                      <EuiFlexItem grow={false}>
-                        <EuiBetaBadge
-                          label={betaBadgeProps.label}
-                          tooltipContent={betaBadgeProps.tooltipContent}
-                        />
-                      </EuiFlexItem>
-                    )
-                  : isExperimental && (
-                      <EuiFlexItem grow={false}>
-                        <EuiBetaBadge
-                          label={technicalPreviewBadgeProps.label}
-                          tooltipContent={technicalPreviewBadgeProps.tooltipContent}
-                        />
-                      </EuiFlexItem>
-                    )}
+                {actionTypeName && isExperimental && (
```

Comment:
Shuldn't we render technical preview badge here?

Reply:
Not sure I follow. These were just a direct revert from the prior PR. 

can you expand on what you mean here?

Reply:
@tomsonpl I think you're confusing with what the badge contents are actually since it is named `betaBadgeProps` even though the betaBadgeProps actually is technical preview badge label and description. I had the same confusion when making the changes earlier. Just to clarify, the current variable names are consistent with `EuiBetaBadge` component name here. 😓 

So in short the revert is indeed showing the tech. preview badge.